snet_cli/commands.py

```diff
@@ -1225,3 +1225,324 @@ def update(self):
+                    for idx, member in enumerate(members):
+                        self._printerr(" - {}".format(member.lower()))
+                if serviceNames:
+                    self._printerr("\nServices:".format(self.args.name))
+                    for idx, service in enumerate(serviceNames):
+                        self._printerr(" - {}".format(service.partition(b"\0")[0].decode("utf-8")))
+                if repositoryNames:
+                    self._printerr("\nRepositories:".format(self.args.name))
+                    for idx, repo in enumerate(repositoryNames):
+                        self._printerr(" - {}".format(repo.partition(b"\0")[0].decode("utf-8")))
+            else:
+                self._printerr("\n{} not registered on network.".format(self.args.name))
+
+        except Exception as e:
+            self._printerr("\nCall error!\nHINT: Check your identity and session.\n")
+            self._error(e)
+
+    def create(self):
+
+        if self.args.name and self.args.members:
+            try:
+                # Check if Organization already exists
+                (found, _, _, _, _, _) = self._getorganizationbyname()
+                if found:
+                    self._printerr("\n{} already exists!\n".format(self.args.name))
+                    return
+
+                members = []
+                members_split = self.args.members.split(',')
+                for idx, m in enumerate(members_split):
+                    members.append(str(m).replace("[", "").replace("]", "").lower())
+
+                registry_contract_def = get_contract_def("Registry")
+                registry_address = self._getstring("registry_at")
+                cmd = ContractCommand(
+                    config=self.config,
+                    args=self.get_contract_argser(
+                        contract_address=registry_address,
+                        contract_function="createOrganization",
+                        contract_def=registry_contract_def)(type_converter("bytes32")(self.args.name),
+                                                            [type_converter("address")(member) for member in members]),
+                    out_f=self.out_f,
+                    err_f=self.err_f,
+                    w3=self.w3,
+                    ident=self.ident)
+                self._printerr("Creating transaction to create organization {}...\n".format(self.args.name))
+                try:
+                    cmd.transact()
+                except Exception as e:
+                    self._printerr("\nTransaction error!\nHINT: Check if {} already exists.\n".format(self.args.name))
+                    self._error(e)
+            except Exception as e:
+                self._printerr("\nTransaction error!\nHINT: Check if address is in hexadecimal with 32 chars.\n".format(self.args.name))
+                self._error(e)
+        else:
+            self._error("Missing params...")
```

Comment:
I would mark required parameters as required when calling ```add_argument``` method. It is not clear which parameters are missing otherwise.